src/js/components/Breadcrumb.js

```diff
@@ -1,251 +1,52 @@
-import classNames from 'classnames';
-import DeepEqual from 'deep-equal';
-import {Link} from 'react-router';
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
 import React, {PropTypes} from 'react';
-import ReactDOM from 'react-dom';
+/* eslint-enable no-unused-vars */
+import {StoreMixin} from 'mesosphere-shared-reactjs';
 
-import IconChevron from './icons/IconChevron';
+import Crumb from './Crumb';
 
-const COLLAPSE_BUFFER = 12;
-const LAST_ITEM_OFFSET = 150; // Difference between scrollWidth and outerWidth
-const PADDED_ICON_WIDTH = 38; // Width of icon + padding
-
-class Breadcrumb extends React.Component {
+class Breadcrumb extends mixin(StoreMixin) {
   constructor() {
```

Comment:
if we're going to extend the `StoreMixin`, can we do `this.store_listeners = [];` so it won't throw an error if the things that extend this don't want to use it?